FundacionParaguaya/MentorApp
# Feature/accessibility

_Commented the following test **'renders the correct date in second Text component'** in 
 FamiliesListItem.test.js_

**To Test**
1. Switch on TalkBack
2. Try opening the logout modal
3. When the modal it should read the content of the modal

4. Switch the language to ES
5. Repeat 2 and 3
